### ISSUE TITLE
Eq 6.10

### ISSUE THREAD

**Issue by mathursuhas** [udlbook/udlbook#152]
Eq 6.10 is the basic equation describing stochastic gradient descent using a batch. The 2nd term is an estimate of the gradient over the training data points in the batch, multiplied by the learning rate. Should this estimate of gradient be a _mean_ across data points in the batch, instead of a _sum_? This shows up at other places in the book too of course since it is a fundamental equation -- for example Eq. 7.3.

The current framing as a sum is technically not wrong - one could argue that the normalizing factor of 1/(batch size) can be absorbed into the learning rate \alpha -- however, doing so would couple the learning rate and the batch size. That is, if one were to change the batch size while experimenting in pytorch, this would effectively change the learning rate too. I'm not sure if you intended this.

**Comment by udlbook** [comment]
Hi -- I agree with your argument and I wish that I had written it in a different way.  However, as you point out, it's not technically wrong.  I'm only really allowed to fix errata in the current version, but I've filed this away to improve in the next edition.
